.github/workflows/image-release.yaml

```diff
@@ -25,3 +25,32 @@ jobs:
       - name: Push 
         run: |
               make docker-push
+      - name: Build & Pushing cloudshell image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: docker/Dockerfile-webtty
+          labels: |-
+            org.opencontainers.image.source=https://github.com/${{ github.repository }}
+            org.opencontainers.image.revision=${{ github.sha }}
+          platforms: linux/amd64,linux/arm64
+          #push: ${{ github.event_name != 'pull_request' }}
+          build-args: |
+            VERSION=${{ steps.get_version.outputs.VERSION }}
+          tags: |-
+             ghcr.io/cloudtty/cloudshell:${{ steps.get_version.outputs.VERSION }}
```

Comment:
Here miss the get_version step and can not get the VERSION from the outputs.

Reply:
fixed